bears-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+const db = knex(knexConfig);
+
+router.post('/', (req,res) => {
+    if (!req.body.name) {
+        res.status(400).json({error:'missing required field: name'});
+    } else {
+    db('bears').insert(req.body)
+        .then(ids => {
+            const id = ids[0];
+            db('bears').where({id: id}).first()
+                .then(bear => {
+                    res.status(201).json(bear);
+                });
+        })
+        .catch(err => res.status(500).json(err));
```

Comment:
note that it doesn't matter as much since you're not sending back a custom error message along with the error, but if you wanted to send back a unique message for a server error in both the insert _and_ findById calls, you would have to chain another .catch() at the end of the nested .then() up above, otherwise all errors will fall through to this one here